.github/scripts/close-invalid-link.cjs

```diff
@@ -35,11 +35,11 @@ module.exports = async ({ github, context }) => {
 
     try {
         const link = issue.data.body.match(/(https?:\/\/github.com\/.*)/)[0];
-        const isValidLink = (await fetch(link)).status === 200;
+        const isValidLink = (await fetch(link)).status >= 200 && (await fetch(link)).status < 300;
```

Comment:
Looks good, perhaps move the fetch call into a variable so that way it’s called once. You can also use `.ok` as well for status 200-299

Reply:
Thanks!